components/wallet-buttonbar.js

```diff
@@ -11,7 +11,7 @@ export default function WalletButtonBar ({
   return (
     <div className={`mt-3 ${className}`}>
       <div className='d-flex justify-content-between'>
-        {isConfigured(wallet) &&
+        {isConfigured(wallet) && wallet.def.requiresConfig &&
```

Comment:
I don't like that WebLN always showed `detach`. This fixes it by only showing it if there's actually something to delete.